orb_models/finetune_utilities/optim.py

```diff
@@ -0,0 +1,132 @@
+import logging
+import re
+from typing import Any, Dict, List, Mapping, Optional, Tuple, Union
+
+import omegaconf
+import torch
+
+Metric = Union[torch.Tensor, int, float]
+MetricCollection = Union[Metric, Mapping[str, Metric]]
+TensorDict = Mapping[str, Optional[torch.Tensor]]
+
+
+OptimizerKwargs = Dict[str, Any]
+
+
+class ParameterGroup(omegaconf.DictConfig):
+    """Param type for specifying Param groups."""
+
+    optimizer_kwargs: OptimizerKwargs
+    filter_string: str
+
+
+ParameterGroups = List[ParameterGroup]
+
+
+def make_parameter_groups(
```

Comment:
Delete this file and replace the logic with a fixed split of the model params into:

- biases and any norm params
- everything else

Then this can be moved to run.py.